src/__tests__/index.node.js

```diff
@@ -39,18 +39,21 @@ const sampleReducer = (state = [], action) => {
 };
 
 test('Instantiation', t => {
-  t.throws(actionEmitterFunc, 'requires the EventEmitter dependency');
+  t.throws(
+    actionEmitterPlugin.provides,
+    'requires the EventEmitter dependency'
+  );
   const mockEventEmitter = getMockEventEmitterFactory();
   t.doesNotThrow(
-    () => actionEmitterFunc(mockEventEmitter),
+    () => actionEmitterPlugin.provides(mockEventEmitter),
```

Comment:
Depending upon how rigorous we want to be with our tests, it would be nice to have a unit test that tests the entry point we expect from users in the wild (i.e. via `app.register`) as opposed to `.provides` which we don't expect anybody to directly access outside of tests.

Reply:
Sounds good, will land and work on a more comprehensive test as well.